v0/destinations/mp/data/MPIdentifyConfig.json

```diff
@@ -2,14 +2,19 @@
   "createdAt": "$created",
   "createdat": "$created",
   "email": "$email",
-  "firstName": "$firstName",
-  "lastName": "$lastName",
-  "firstname": "$firstName",
-  "lastname": "$lastName",
-  "first_name": "$firstName",
-  "last_name": "$lastName",
+  "firstName": "$first_name",
+  "lastName": "$last_name",
+  "firstname": "$first_name",
+  "lastname": "$last_name",
+  "first_name": "$first_name",
+  "last_name": "$last_name",
   "name": "$name",
   "username": "$username",
   "phone": "$phone",
-  "avatar": "$avatar"
+  "avatar": "$avatar",
+  "city": "$city",
+  "country": "$country_code",
+  "state": "$region",
+  "geoSource": "$geo_source",
+  "unsubscribed": "$unsubscribed"
```

Comment:
Most of the default properties are not mapped yet.
CC @utsabc

Reply:
I am following https://help.mixpanel.com/hc/en-us/articles/115004708186 this. I mapped everything which is in this link.

Reply:
The default properties the customer has pointed out are not mapped (see the original thread)